Practitioner/UKCore-Practitioner-LucyHale-Example.json

```diff
@@ -1,6 +1,6 @@
 {
 	"resourceType": "Practitioner",
-	"id": "6e1a8ea3-3a59-40aa-b943-5c896a30e43a",
```

Comment:
May need to look at references used in some Bundles if we are switching UUIDs for human readable IDs across the board.